### PR TITLE
* Make npm quieter

### PR DESCRIPTION
**Pull request by pranavkm**
- Run npm install in parallel

### CODE REVIEW

**Comment by pranavkm** [comment]
cc @dougbu

File: build/shade/_k-standard-goals.shade [hunk 1]
```diff
@@ -37,9 +38,12 @@ default PACKAGELIST_JSON_FILENAME = 'NuGetPackageVerifier.json'
...
 
 #restore-npm-modules
-  -// Find all dirs that contain a package.json file
-  var npmDirs = '${GetDirectoriesContaining(Directory.GetCurrentDirectory(), "package.json")}'
-  npm npmCommand='install ${E("KOREBUILD_NPM_INSTALL_OPTIONS")}' each='var npmDir in npmDirs'
+  @{
+    // Find all dirs that contain a package.json file
+    var npmDirs = GetDirectoriesContaining(Directory.GetCurrentDirectory(), "package.json").ToArray();
+    var npmOptions = E("KOREBUILD_NPM_INSTALL_OPTIONS") ?? " --quiet";
+    Parallel.ForEach(npmDirs, npmDir => Npm("install " + npmOptions, npmDir));
```
Comment (dougbu) [inline build/shade/_k-standard-goals.shade:45]: Does this really take so long that it's worth turning the output (when `--quiet`) into strudel?

File: build/shade/_k-standard-goals.shade [hunk 2]
```diff
@@ -233,7 +237,7 @@ default PACKAGELIST_JSON_FILENAME = 'NuGetPackageVerifier.json'
   @{
     AddToE("KOREBUILD_DOTNET_RESTORE_OPTIONS", "--verbosity minimal");
     AddToE("KOREBUILD_BOWER_INSTALL_OPTIONS", "--quiet");
-    AddToE("KOREBUILD_NPM_INSTALL_OPTIONS", "--quiet");
+    AddToE("KOREBUILD_NPM_INSTALL_OPTIONS", "--silent");
```
Comment (dougbu) [inline build/shade/_k-standard-goals.shade:240]: :+1:

File: build/shade/_k-standard-goals.shade (hunk 1 above)
Comment (pranavkm) [inline build/shade/_k-standard-goals.shade:45]: It takes about a 1minute 45 on the CI for Mvc. Anything to speed it up.

Comment (dougbu) [inline build/shade/_k-standard-goals.shade:45]: :ok:

**Comment by dougbu** [comment]
:shipit:

**Pull request merged**